### PR TITLE
Feature/public profile

### PR DESCRIPTION
**Pull request by rej156**

### CODE REVIEW

File: packages/kauri-components/components/Separators/Separator.re [hunk 1]
```diff
@@ -51,14 +33,14 @@ let getColorStyle = color =>
   | White => Styles.whiteColor
   };
 
-let make = (~direction, ~color=White, ~my=?, _children) => {
+let make = (~direction, ~color=White, ~my=?, ~mt=?, _children) => {
```
Comment (davodesign84) [inline packages/kauri-components/components/Separators/Separator.re]: what are my and mt? Can we make them more explicit?

File: packages/kauri-web/components/connections/PublicProfile/ProfileAvatar_Connection.re [hunk 2]
```diff
@@ -0,0 +1,40 @@
+let (|?) = (a, b) =>
```
Comment (davodesign84) [inline packages/kauri-web/components/connections/PublicProfile/ProfileAvatar_Connection.re]: really don't like these... but if they need to be there shall we export them in a file?

File: packages/kauri-web/components/containers/Navbar/View.js [hunk 3]
```diff
@@ -132,7 +132,7 @@ export default class Navbar extends React.Component {
           </Link>
         </StyledMenuItem>
 
-        <ArticleSearchbar />
```
Comment (davodesign84) [inline packages/kauri-web/components/containers/Navbar/View.js]: Please rebase this :D

File: packages/kauri-web/lib/ReasonMLTypings/Resources/Article_Resource.re [hunk 4]
```diff
@@ -0,0 +1,93 @@
+let (|?) = (a, b) =>
```
Comment (davodesign84) [inline packages/kauri-web/lib/ReasonMLTypings/Resources/Article_Resource.re]: Definitely need to export these

**Comment by rej156** [comment]
Ammended

**Pull request merged**